exporter/signalfxexporter/dimensions/metadata_test.go

```diff
@@ -129,14 +129,14 @@ func TestGetDimensionUpdateFromMetadata(t *testing.T) {
 				Name:  "name",
 				Value: "val",
 				Properties: getMapToPointers(map[string]string{
-					"prope_rty1": "value1",
-					"prope_rty2": "",
+					"prope/rty1": "value1",
+					"prope.rty2": "",
 					"prope_rty3": "value33",
-					"prope_rty4": "",
+					"prope.rty4": "",
 				}),
 				Tags: map[string]bool{
-					"ta_g1": true,
-					"ta_g2": false,
+					"ta.g1": true,
```

Comment:
Add test to cover behavior when translator is not `nil`.

Reply:
There is one test with a translator provided already https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/1220/files#diff-7a8eea08662885f9f00790a23bae37b94342be2905692e8c4bc37884ab996f17R164
Or you want another specific use case?